frontend/packages/operator-lifecycle-manager/src/components/modals/uninstall-operator-modal.tsx

```diff
@@ -34,7 +53,18 @@ export const UninstallOperatorModal: React.FC<UninstallOperatorModalProps> = ({
   subscription,
 }) => {
   const { t } = useTranslation();
-  const [handlePromise, inProgress, errorMessage] = usePromiseHandler();
+  const [
+    handleOperatorUninstallPromise,
+    operatorUninstallInProgress,
+    operatorUninstallErrorMessage,
+  ] = usePromiseHandler();
+  const [operatorUninstallFinished, setOperatorUninstallFinished] = React.useState(false);
+  const [isDeleteAllOperandsSelected, setIsDeleteAllOperandsSelected] = React.useState(false);
```

Comment:
I was confused by `DeleteAll...` reading the code because it made me think you could select individual operands. If delete all is the only option, I would make this

```suggestion
  const [deleteOperands, setDeleteOperands] = React.useState(false);
```

Reply:
renamed